jbmc/regression/jbmc/context-include-exclude/test_excluded_deleted_original_body.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+ExcludedProperties.class
+--context-exclude org.cprover.other --function ExcludedProperties.runtimeReturnType
+^EXIT=10$
+^SIGNAL=0$
+.* line 30 assertion at file ExcludedProperties.java line 30.*: FAILURE
```

Comment:
:question: there is no assertion line 30, as far as I can see, I mean 28?

Reply:
As discussed, this was a result of a late clang-format run (30 was correct before clang-format) and should be fixed now.